lib/home.dart

```diff
@@ -54,19 +51,19 @@ class _HomeState extends State<Home> with SingleTickerProviderStateMixin {
                   children: [
                     Container(
                       child: Image.asset(
-                        (_themeChanger.lightDark)?'assets/logoWideBlack.png':'assets/logoWideWhite.png',
+                        (_themeChanger.lightDark)
+                            ? 'assets/logoWideBlack.png'
+                            : 'assets/logoWideWhite.png',
                       ),
                       width: 160,
                     ),
                     Switch(
                         value: _themeChanger.lightDark,
-                        onChanged: (lightDark){
-                          if(lightDark)
-                         _themeChanger.setTheme(ThemeData.light());
-                         else
-                          _themeChanger.setTheme(ThemeData.dark());
-
-
```

Comment:
This PR is all about contributors page. Try to address single issue in a single PR.

You can always send multiple PRs for multiple issues. ;)

Reply:
okay i will keep in mind for next time